health/micro-ui/web/micro-ui-internals/packages/modules/health-dss/src/components/RichSummary.js

```
@@ -0,0 +1,161 @@
+      {chartData?.headerSymbol !== "percentage" ? (
+        <div className="digit-metric-data-header">
+          {`${Digit.Utils.dss.formatter(chartData?.headerValue, chartData?.headerSymbol, value?.denomination, true, t)}`}
+        </div>
+      ) : (
+        <div style={{ width: "80%", margin: "auto" }}>
+          <CircularProgressBar data={chartData} />
+        </div>
+      )}
+      <div className={`tooltip`}>
+        <div className="digit-dss-metric-card-text">{typeof name == "string" && name}</div>
+        {Array.isArray(name) && name?.filter((ele) => ele)?.map((ele) => <div style={{ whiteSpace: "pre" }}>{ele}</div>)}
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)